### DISCUSSION TITLE
Why does the documentation suggest installing Serverless globally by default?

### DISCUSSION THREAD

**Discussion by klejejs** [serverless/serverless#12020] [Q&A]
From my experience, there can be different projects with different configuration file ages, so not all configurations support all serverless versions, and that often creates issues when someone picks up an old project and just wants to deploy it. To mitigate this issue, we install serverless in each project separately, each project can then have its own serverless version, and it always works with that project's configuration.
Thus, my question, why does the documentation suggest installing serverless globally by default, instead of installing it in a project?

**Comment by ajunx8** [comment]
I have the same question at the moment.

**Reply by ajunx8** [reply]
I read that if you have it installed in a project, all your serverless commands need to be prefixed with `npm run` 
I could see how that would be annoying but I'm not sure if that's the primary reason for installing it globally.
